ipyvolume/pylab.py

```diff
@@ -370,16 +370,19 @@ def plot(x, y, z, color=default_color, **kwargs):
     """
     fig = gcf()
     _grow_limits(x, y, z)
-    scatter = ipv.Scatter(x=x, y=y, z=z, color=color, color_selected=None,
-        size_selected=1, size=1,
-                             connected=True, visible_markers=False, visible_lines=True, **kwargs)
-    fig.scatters = fig.scatters + [scatter]
-    return scatter
+    defaults = dict(visible_lines=True, color_selected=None, size_selected=1,
+                    size=1, connected=True, visible_markers=False)
+    kwargs = dict(defaults, **kwargs)
+    scatter_hdl = ipv.Scatter(x=x, y=y, z=z, color=color, **kwargs)
```

Comment:
Why rename do scatter_hdl? I don't know what hdl means.

Reply:
two reasons -

1. while it technically works there is a name clash, `scatter` is a method as well. so there is a local vs global clash that i do not like.
2. `_hdl` stands for "handle" since it's really a pointer to the object.

Reply:
fair point, but I don't abrvs that aren't std ;). I'd rather see s = ipv.Scatter(..), and for python it's always by reference so _hdl doesn't add information, except indeed preventing a possible name collision. my preference goes to `s` or `scatter_object`